jest.config.js

```diff
@@ -177,10 +177,10 @@ module.exports = {
   // transform: undefined,
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-  // transformIgnorePatterns: [
-  //   "/node_modules/",
+  transformIgnorePatterns: [
+    "/node_modules/@cloudinary/url-gen/\\.(js|ts|tsx)$",
```

Comment:
same as the above comment. it feels like this is glossing over something deeper. do we know why we need to add this?

Reply:
this is the issue that I get when I remove the fix
<img width="922" alt="Screenshot 2022-05-24 at 14 44 45" src="https://user-images.githubusercontent.com/102802882/170050601-72f61596-a5ec-48af-9db1-d45dd413ba11.png">

Reply:
ah k. makes sense. probably not worth digging deeper.